components/TabBar/TabBar.tsx

```diff
@@ -7,60 +7,65 @@ import { peach } from "../../assets/colors/colors"
 import { Ionicons } from "@expo/vector-icons"
 import { globalStyles } from "../../assets/global/globalStyles"
 
-type IconName = "home" | "home-outline" | "duplicate" | "duplicate-outline" | "compass" | "compass-outline"
+type IconName =
+  | "globe"
+  | "globe-outline"
+  | "magnet"
+  | "magnet-outline"
+  | "compass"
+  | "compass-outline"
 
-export const TabBar = ({ state, descriptors, navigation }: BottomTabBarProps) => {
+export const TabBar = ({
+  state,
+  descriptors,
+  navigation,
+}: BottomTabBarProps) => {
   const insets = useSafeAreaInsets()
 
   return (
     <View style={[styles.tabBar, { paddingBottom: insets.bottom }]}>
       {state.routes.map((route, index) => {
         const { options } = descriptors[route.key]
         const label =
-          options.tabBarLabel !== undefined
-            ? options.tabBarLabel.toString()
-            : options.title !== undefined
-              ? options.title
-              : route.name
-
+          options.tabBarLabel?.toString() ?? options.title ?? route.name
```

Comment:
Comments wouldn't hurt here!

Reply:
It is a change suggested by Alberto. It is explained in the discussions here it has to do with how JS syntax works now